packages/template-ui/src/views/Home.vue

```diff
@@ -117,7 +108,13 @@ export default {
       this.fetchCarouselNodes(),
       this.fetchContentNodes(),
       this.fetchSectionNodes(),
-    ]);
+    ]).then(() => {
+      if (this.displayHeroContent) {
+        const [first] = this.contentNodes.nodes;
+        // Showing the first node
+        this.goToContent(first);
+      }
```

Comment:
It looks like `displayHeroContent` isn't really doing what it says any more, so we should rename it :)

Also, I wonder what happens here if a channel has more than one content node? Perhaps we can change `displayHeroContent` to something like `defaultContentNode="12345abcdef"`, or make this only happen if `this.contentNodes.nodes.length === 1`?

Reply:
It shows just the first content node if there's more than one. I think it's simpler than the `defaultContentNode` configuration, but maybe it's a good idea to add that option and just fallback to the first if it's not set, so this can work correctly even if the content id changes in the channel.

Reply:
I've added a commit to rename that option to the proposed one.